Jenkinsfile

```diff
@@ -70,9 +70,7 @@ def testStage(name, body) {
 }
 
 node {
-    env.BRANCH_NAME = env.BRANCH_NAME ?: 'master'
-    String gitref = env.CHANGE_ID ? "pr/${env.CHANGE_ID}" : env.BRANCH_NAME
-    def fhPipelineLibrary = library("fh-pipeline-library@${gitref}")
+    def fhPipelineLibrary = library("fh-pipeline-library@${env.BRANCH_NAME ?: 'master'}")
```

Comment:
I'm not sure we ever want to silently fall back to master here. @mikenairn should I just change it to be just `env.BRANCH_NAME`?

Reply:
Yeah it probably doesn't make much sense to fallback to another branch here.